### DISCUSSION TITLE
Feature Request: Choose between available resolutions

### DISCUSSION THREAD

**Discussion by petenun** [anidl/multi-downloader-nx#717] [Ideas]
It would be nice if there was an option to choose between available video resolutions (ie 720p, 1080p, etc) for downloads.

**Comment by Jaynator495** [comment]
You can do that with -q. I really should consider an alternative flag to set the desired resolution though. Also this should be posted in issues

**Comment by petenun** [comment]
Another question: Looking at the documentation for -q Quality Levels, it appears that accepted values for -q are 1080p, 720p, etc,, yet the only accepted values are numeric 1, 2, 3, etc. What is the numeric value that corresponds to 720p?

[Note: I originally didn't post in Issues since I felt this was a lack of understanding on my part, not a bug or defect in the product.]

**Reply by petenun** [reply]
Via trial and error, I was able to determine that Quality 5 equates to 720p, so problem solved. 
It would be a nice feature request to allow "real" values for the -q setting such as 480p, 720p, 1080p.
